quelpa.el

```diff
@@ -92,6 +92,11 @@ the `:upgrade' argument."
   :group 'quelpa
   :type 'string)
 
+(defcustom quelpa-melpa-recipe-dirs `(,(expand-file-name "package-build/recipes" quelpa-build-dir))
```

Comment:
use

```cl
(list (expand-file-name "package-build/recipes" quelpa-build-dir))
```
` is just used as a convenience if you need to evaluate particular expressions inside a list